### PR TITLE
Update README.md with missing build dependency.

### PR DESCRIPTION
**Pull request by ILMTitan**
Add the nyc as a global build dependency.

Reorder the sections to: Install, Docs, Support, Build, Contribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
-  - [Node.js][Node]. Download and install from the website.
-  - [npm][npm]. Should be installed with the Node.js install.
-  - [TypeScript][TypeScript]. Install with `npm install -g typescript`
-  - [tfx][tfs-cli] The tfs cli. Install with `npm install -g tfx-cli`
+  - [PowerShell][PowerShell]: Installed by default on modern windows platforms.
+  - [Node.js][Node]: Download and install from the website.
+  - [npm][npm]: Should be installed with the Node.js install.
+  - [TypeScript][TypeScript]: Install with `npm install -g typescript`
+  - [tfx][tfs-cli]: The tfs cli. Install with `npm install -g tfx-cli`
+  - [nyc][nyc]: The Istanbul code coverage tool. Install with `npm install -g nyc`
```
Comment (iantalarico) [inline README.md]: We also need mocha and ts-node to run the tests

**Review by iantalarico — APPROVED** [review]
I think you need to add a few missing deps otherwise LGTM.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
 
 ### Prerequisites
 
-  - [PowerShell][PowerShell]. Installed by default on modern windows platforms.
-  - [Node.js][Node]. Download and install from the website.
-  - [npm][npm]. Should be installed with the Node.js install.
-  - [TypeScript][TypeScript]. Install with `npm install -g typescript`
-  - [tfx][tfs-cli] The tfs cli. Install with `npm install -g tfx-cli`
+  - [PowerShell][PowerShell]: Installed by default on modern windows platforms.
+  - [Node.js][Node]: Download and install from the website.
```
Comment (iantalarico) [inline README.md:36]: Actually one more thing.  This should probably mention 6.x must be used

**Review by iantalarico — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ILMTitan) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (ILMTitan) [reply README.md:36]: I will fix this dependency issue in a separate PR.

**Pull request merged**